aws/resource_aws_globalaccelerator_endpoint_group.go

```diff
@@ -87,6 +87,10 @@ func resourceAwsGlobalAcceleratorEndpointGroup() *schema.Resource {
 							Type:     schema.TypeInt,
 							Optional: true,
 						},
+						"client_ip_preservation_enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
```

Comment:
I think we need to add `Computed: true,` here as the flag is implicitly enabled for internal ALB or instances: https://docs.aws.amazon.com/global-accelerator/latest/dg/preserve-client-ip-address.how-to-enable-preservation.html.